File: 2436-make-array-zero-by-subtracting-equal-amounts/make-array-zero-by-subtracting-equal-amounts.py
class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        nums.sort()
        n = len(nums)
        cnt = 1
        if(nums[0] == 0):
            cnt = 0
        for i in range(n-1):
            if(nums[i+1] - nums[i] > 0):
                cnt+=1
        return cnt

File: 2436-make-array-zero-by-subtracting-equal-amounts/make-array-zero-by-subtracting-equal-amounts.java
class Solution {
    public int minimumOperations(int[] nums) {
        Arrays.sort(nums);
        int n=nums.length;
        int cnt=1;
        if(nums[0] == 0) cnt = 0;
        for(int i=0; i<n-1; i++){
            if(nums[i+1] - nums[i] > 0) cnt++;
        }
        return cnt;
    }
}